### PR TITLE
Mjm/master

### PR DESCRIPTION
**Pull request by MisShellyMac**
Here is the silver level attempt for the random menu generator. I was thinking about trying to use the gem colorize to make it so that each type of category was a different color but I never ended up completing that.

### CODE REVIEW

File: menu_silver.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+styles = ['fried', 'steamed', 'baked', 'braised', 'roasted', 'boiled', 'pressure-cooked', 'broiled',
+  'freeze-dried', 'dehydrated', 'carmelized', 'stewed', 'pickled', 'seared',  'bar-b-qued', 'marinated',
+  'candied']
+
+foods = ['chicken','pork', 'shrimp', 'tilapia', 'beef', 'turkey', 'meatloaf', "pie", 'cookie', 'ice cream',
+  'donut', 'cake','mashed potatoes', 'hamburger', 'sausage', 'bananas', 'crab' ]
+
+prng = Random.new
+
+(0..foods.length-1).each do |i|
```
Comment (GreyKn) [inline menu_silver.rb:13]: Cool, you used an loop type has built in incrementation. :+1: 

Another way you could have done this is:

```
(foods.length).times do |i|
  ...
end
```

Comment (GreyKn) [inline menu_silver.rb:13]: `i` as a counter is a norm, but let's be better than normal and use something more descriptive.

File: menu_silver.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+adjectives = ['hot', 'cold', 'creamy', 'soft', 'hard', 'crunchy', 'warm', 'spicy', 'gelatinous',
```
Comment (GreyKn) [inline menu_silver.rb:1]: When you're defining a bunch of strings in an array, it's hard to keep formatting, spacing, quote styling the same, and not leaving dangling entries.

One way around it is to use `%w` or `%W`.

So for example, you could rewrite adjectives as:

```
adjectives = %w[ hot cold creamy soft hard crunchy warm 
                 spicy gelatinous chewy crispy juicy]
 => ["hot", "cold", "creamy", "soft", "hard", "crunchy", "warm", "spicy", "gelatinous", "chewy", "crispy", "juicy"]
```

And voila, it'll separate all the words at the spaces and do all the quoting for you.

But what if there are spaces in your word, like in foods? You can use `%W` with escapes (`\s` is space)

```
foods = %W[ chicken mashed\spotatoes  hamburger]
 => ["chicken", "mashed potatoes", "hamburger"] 
```

**Comment by GreyKn** [comment]
Hi Shelly,

Good job on the assignment. My suggestion is that even though it's a small assignment, to use more descriptive variable names or add comments to define what shorthand variables are for.

Let me know if have any questions.

File: menu_silver.rb (hunk 2 above)
Comment (MisShellyMac) [inline menu_silver.rb:1]: thanks for the note about this! We hadn't really learned that yet but I definitely see how it would be helpful!

**Comment by MisShellyMac** [comment]
thank you for the code review!

**Pull request merged**